site/app/templates/polls/NewPollPage.twig

```diff
@@ -79,23 +89,39 @@
             </label>
             <br/><br/>
 
+            <span class="option-title" id="poll-question-type-label">Question Type:</span>
+            <label for="poll-type-single-response">
+                {% if poll.getQuestionType() == 'single-response'%}
+                    <input type="radio" id="poll-type-single-response" name="question-type" value="single-response" checked="">Single response (radio buttons)</input>
+                {% else %}
+                    <input type="radio" id="poll-type-single-response" name="question-type" value="single-response">Single response (radio buttons)</input>
+                {% endif %}
+            </label>
+            <label for="poll-type-multiple-response">
+                {% if poll.getQuestionType() == 'single-response'%}
+                    <input type="radio" id="poll-type-multiple-response" name="question-type" value="multiple-response">Multiple response (checkboxes)</input>
+                {% else %}
+                    <input type="radio" id="poll-type-multiple-response" name="question-type" value="multiple-response" checked="">Multiple response (checkboxes)</input>
+                {% endif %}
```

Comment:
Rather than needing to write this 2 times you could do something like this:

`<input type="radio" id="poll-type-single-response" name="question-type" value="single-response" {% if poll.getQuestionType() == 'single-response' %} checked {% endif %}>Single response...</input>`

Reply:
Cool! I didn't know this was possible. I'll change this